lib/src/Uncertainty/Algorithm/Sensitivity/MauntzKucherenkoSensitivityAlgorithm.cxx

```diff
@@ -100,10 +101,7 @@ Sample MauntzKucherenkoSensitivityAlgorithm::computeIndices(const Sample & sampl
       for (UnsignedInteger q = 0; q < outputDimension; ++q)
       {
         varianceI(q, p) =  (yEDotyB[q] - yADotyB[q]) / (size - 1.0);
-        // Vti = Var - V_{-i}
-        // \sum_{k} yA[k] * yA[k] - yA[k]*yE[k]
-        // yA[k] * yA[k]  = sigma_a^2 + muA^2
-        VTi(q, p) = referenceVariance_[q] + (size * muA[q] *  muA[q] - yEDotyA[q]) / (size - 1.0);
+        VTi(q, p) = (yADotyA[q] - yEDotyA[q]) / (size - 1.0);
```

Comment:
I am not sure why you dropped `referenceVariance_[q]`?
```suggestion
        VTi(q, p) = referenceVariance_[q] + (yADotyA[q] - yEDotyA[q]) / (size - 1.0);
```

Reply:
Because we apply the exact formula!

Reply:
OK. I think the explanation given in your [Discourse post](https://openturns.discourse.group/t/mauntz-kucherenko-formula/68/3) should also be given in the doc, otherwise we may forget and rediscuss this again in the future.